projects/storefrontapp-e2e-cypress/cypress/e2e/regression/cart/cart.e2e.cy.ts

```diff
@@ -20,8 +20,7 @@ describe('Cart', () => {
     });
 
     context('Registered user', () => {
-      before(() => {
-        cy.window().then((win) => win.sessionStorage.clear());
```

Comment:
not  needed as testIsolation clear cookies/localStorage/sessionStorage before 'before' and 'beforeEach' hooks.